circuits/cpp/barretenberg/cpp/src/barretenberg/honk/sumcheck/sumcheck.hpp

```diff
@@ -68,27 +64,20 @@ template <typename Flavor, class Transcript> class Sumcheck {
     PartiallyEvaluatedMultivariates partially_evaluated_polynomials;
 
     // prover instantiates sumcheck with circuit size and a prover transcript
-    Sumcheck(size_t multivariate_n, ProverTranscript<FF>& transcript)
+    SumcheckProver(size_t multivariate_n, ProverTranscript<FF>& transcript)
         : transcript(transcript)
         , multivariate_n(multivariate_n)
         , multivariate_d(numeric::get_msb(multivariate_n))
         , round(multivariate_n)
         , partially_evaluated_polynomials(multivariate_n){};
 
-    // verifier instantiates sumcheck with circuit size and a verifier transcript
-    explicit Sumcheck(size_t multivariate_n, VerifierTranscript<FF>& transcript)
-        : transcript(transcript)
-        , multivariate_n(multivariate_n)
-        , multivariate_d(numeric::get_msb(multivariate_n))
-        , round(){};
-
     /**
      * @brief Compute univariate restriction place in transcript, generate challenge, partially evaluate,... repeat
      * until final round, then compute multivariate evaluations and place in transcript.
      *
      * @details
      */
-    SumcheckOutput<Flavor> execute_prover(
+    SumcheckOutput<Flavor> execute(
```

Comment:
why do we call this `execute` instead of `prove`?

Reply:
It was originally `execute_prover` so I just removed the redundancy. I think "execute" is natural in the sense of 'executing the sumcheck prover protocol'. Again, I dont feel strongly here.

Reply:
changed to prove and verify